### PR TITLE
[IOS-2863]Add super token support for MoPub adapter

### PR DESCRIPTION
**Pull request by felixzhang0703**

### CODE REVIEW

File: Vungle/VungleRouter.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 #import <Foundation/Foundation.h>
 #import <VungleSDK/VungleSDK.h>
+#import <VungleSDK/VungleSDKHeaderBidding.h>
```
Comment (YueVungle) [inline Vungle/VungleRouter.h]: Could move the header file to  "VungleRouter.m"  file, thanks. :)

File: Vungle/VungleRouter.h [hunk 2]
```diff
@@ -40,6 +41,7 @@ extern const CGSize kVNGLeaderboardBannerSize;
 - (void)requestRewardedVideoAdWithCustomEventInfo:(NSDictionary *)info delegate:(id<VungleRouterDelegate>)delegate;
 - (void)requestBannerAdWithCustomEventInfo:(NSDictionary *)info size:(CGSize)size delegate:(id<VungleRouterDelegate>)delegate;
 - (BOOL)isAdAvailableForPlacementId:(NSString *)placementId;
+- (NSString*)currentSuperToken;
```
Comment (YueVungle) [inline Vungle/VungleRouter.h]: Should `- (NSString *)`, there is a space between `NSString` and `*`, thanks. :)

File: Vungle/VungleRouter.m [hunk 3]
```diff
@@ -315,6 +315,14 @@ - (BOOL)isBannerAdAvailableForPlacementId:(NSString *)placementId size:(CGSize)s
     return [[VungleSDK sharedSDK] isAdCachedForPlacementID:placementId withSize:[self getVungleBannerAdSizeType:size]];
 }
 
+- (NSString *)currentSuperToken {
+
```
Comment (YueVungle) [inline Vungle/VungleRouter.m]: There is an extra line, thanks. :)

Comment (felixzhang0703) [reply Vungle/VungleRouter.m]: Resolved.

File: Vungle/VungleRouter.h (hunk 2 above)
Comment (felixzhang0703) [reply Vungle/VungleRouter.h]: Resolved.

File: Vungle/VungleRouter.h (hunk 1 above)
Comment (felixzhang0703) [reply Vungle/VungleRouter.h]: Resolved.

File: Vungle/VungleRouter.m [hunk 4]
```diff
@@ -14,6 +14,7 @@
 #endif
 #import "VungleInstanceMediationSettings.h"
 #import "VungleAdapterConfiguration.h"
+#import <VungleSDK/VungleSDKHeaderBidding.h>
```
Comment (YueVungle) [inline Vungle/VungleRouter.m]: It' better to put `<VungleSDK/VungleSDKHeaderBidding.h>` before `#import "VungleInstanceMediationSettings.h"` to follow MoPub code style guidance, thanks a lot. :)

Comment (felixzhang0703) [reply Vungle/VungleRouter.m]: Resolved.

**Comment by felixzhang0703** [comment]
Create a new PR based on 6.6.x_pre_release branch.
Close this one.

**Pull request closed without merging**